toplevel/toploop.ml

```diff
@@ -109,6 +109,7 @@ let load_file = load_file false
 (* Execute a script.  If [name] is "", read the script from stdin. *)
 
 let run_script ppf name args =
+  Clflags.debug := true;
```

Comment:
Is this related to the bug fix or an independent change?

Reply:
Without this, the toplevel wouldn't collect debug information and show no line numbers after an exception.

Reply:
I still don't understand.  Is this change visible to users?  If so it should have been discussed as part of this PR and it should be described as such in the Changes file.

Reply:
I understand this as a minor fix to an inconsistency in the toplevel code: `Toploop.loop`, called by interactive usage of the toplevel, sets `Clflags.debug := true` on entry, but (until this change) `ocaml foo.ml` would behave differently. With this change, both Toploop entry points from Topmain do the same thing.

Reply:
That will affect the output of shebang ocaml scripts, which is a known area for surprising bug reports (cf. the still-smouldering, if closed, stuff around #2155). That sounds worth an explicit entry in Changes, therefore?